src/config.ts

```diff
@@ -134,6 +134,26 @@ const pontusxConfig: LayerConfig = {
   type: RuntimeTypes.Evm,
 }
 
+const pontusxTestConfig: LayerConfig = {
```

Comment:
I am not super familiar with PontusX, but why we cannot have one layer with mainnet, testnet and devenet ?

Reply:
> The decision (between deltaDao and Jernej) was that they want to run both their devnet and testnet on our testnet.

Also, they have already deployed their paratimes (both on testnet) accordingly. 

So really, there is not much we can do about this in Explorer. (How we implement the layer selection is another matter, but that will handle that on the pontusx branch.)